### PR TITLE
Integrate SSAI namespace

### PR DESCRIPTION
**Pull request by rolandkakonyi**
## Description

This PR integrates the `SsaiApi` class into the public API of `ConvivaAnalytics`.

## Changes

Added new property to access the SSAI API via `convivaAnalytics.ssai`.

## Checklist
- [x] 🗒 `CHANGELOG` entry

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -142,6 +144,41 @@ convivaAnalytics.updateContentMetadata(metadata);
...
+convivaAnalytics.ssai.reportAdFinished()
+convivaAnalytics.ssai.reportAdBreakFinished()
+```
+
+In addition to the metadata provided in the `SsaiAdInfo` object at ad start, the following metadata will be auto collected from the main content metadata:
+
+- Stream URL
+- Asset Name
+- Is live flag
+- Defautl resource
```
Comment (strangesource) [inline README.md:173]:
```suggestion
- Default resource
```

File: README.md [hunk 2]
```diff
@@ -142,6 +144,41 @@ convivaAnalytics.updateContentMetadata(metadata);
 
 Those values will be cleaned up after the session is closed.
 
+### Server Side Ad Tracking
+
+In order to track server side ads you can use the methods provided in `SsaiApi` which can be accessed via `ConvivaAnalytics.ssai`.
+The following example shows basic server side ad tracking:
+
+```swift
```
Comment (strangesource) [inline README.md:152]:
```suggestion
```ts
```

**Review by strangesource — APPROVED** [review]
Nice, just some NIT.

File: README.md (hunk 2 above)
Comment (rolandkakonyi) [reply README.md:152]: oops, thanks fixed it by accepting the suggestion!

File: README.md (hunk 1 above)
Comment (rolandkakonyi) [reply README.md:173]: oops, thanks fixed it by accepting the suggestion!

**Pull request merged**